### ISSUE TITLE
Can as-pect mock functions, and expect toHaveBeenCalled?

### ISSUE THREAD

**Issue by xiong-pro** [jtenner/as-pect#361]
Hi, how can i test a function (A), which will call another function (B). I expect the B will be called.

**Comment by jtenner** [comment]
Not yet. This is something I plan on looking into.

**Comment by jtenner** [comment]
I don't believe closures will be supported anytime soon. On top of this, we don't keep track of when functions are called, what function indexes are used, and this would be a feature that AssemblyScript is not good at providing. I think we should close this issue and mark as "wontfix." Perhaps some day we can add this feature.
